app/services/user.js

```diff
@@ -12,3 +12,13 @@ exports.getUsers = ({ page }, next) =>
   User.findAll({ limit: DEFAULT_PAGE_LIMIT, offset: page ? (page - 1) * DEFAULT_PAGE_LIMIT : 0 }).catch(err =>
     next(databaseError(err))
   );
+
+exports.createOrUpdateToAdminUser = ({ email, ...otherUserData }, next) =>
```

Comment:
no need to send the next here, just throw the error and the catch in the controller should get it